examples/poc01_1/poc01_1.go

```diff
@@ -47,14 +47,14 @@ func (poccc *POC1_1_Chaincode) Invoke(stub shim.ChaincodeStubInterface) peer.Res
 	switch funcName {
 	case constants.AddAssetType:
 		return poccc.addAssetType(stub, args)
-	case constants.Manufacture:
-		return poccc.manufacture(stub, args)
+	case constants.Create:
```

Comment:
I think this should be Manufacture. The "endpoint" should be manufacture, even the chaincode method might be manufacture, but it needs to be calling the abstract `create` method of the base chaincode